packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFDataConverters.h

```diff
@@ -0,0 +1,19 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "FWFGeneratedWebKitApis.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * Converts an FWFNSUrlRequestData to an NSURLRequest.
+ *
+ * @param data The data object containing information to create an NSURLRequest.
+ *
+ * @return An NSURLRequest or nil if data could not be converted.
+ */
+extern NSURLRequest* _Nullable FWFNSURLRequestFromRequestData(
```

Comment:
It wouldn't hurt to have some dedicated unit tests for this conversion function. They should be trivial, but would avoid failures potentially showing up at a higher level where they would be harder to pinpoint.